lib/fog/compute/kubevirt/models/vm_base.rb

```diff
@@ -68,6 +78,39 @@ def stop(options = {})
           service.update_vm(vm)
         end
       end
+
+      class VmNic
```

Comment:
Those classes could be useful for vms as well. Can we create them in separate files?